content/tutorial/tutorial.md

```diff
@@ -448,19 +454,19 @@ class Square extends React.Component {
 }
 ```
 
-When a Square is clicked, the `onClick` function provided by the Board is called. Here's a review of how this is achieved:
+जब एक स्क्वायर पर क्लिक किया जाता है, तो Board द्वारा प्रदान किया गया `onClick` फ़ंक्शन कॉल होता है। यहां बताया गया है कि यह कैसे हासिल किया जाता है:
 
-1. The `onClick` prop on the built-in DOM `<button>` component tells React to set up a click event listener.
-2. When the button is clicked, React will call the `onClick` event handler that is defined in Square's `render()` method.
-3. This event handler calls `this.props.onClick()`. The Square's `onClick` prop was specified by the Board.
-4. Since the Board passed `onClick={() => this.handleClick(i)}` to Square, the Square calls `this.handleClick(i)` when clicked.
-5. We have not defined the `handleClick()` method yet, so our code crashes. If you click a square now, you should see a red error screen saying something like "this.handleClick is not a function".
+1. अंतर्निहित 'DOM' पर `onClick` prop `<button>` कौम्पोनॅन्ट क्लिक इवेंट लिस्टनेर स्थापित करने के लिए React को बताता है।
```

Comment:
अंतर्निहित should not be translated maybe?

Reply:
Also, the statement does not make sense when compared with the original line

Reply:
Agreed. Hindi line has a different sense. There I can't figure out that onClick prop button is on button component.

Reply:
> अंतर्निहित should not be translated maybe?

`बिल्ट-इन` ?

Reply:
`बिल्ट-इन DOM के `<button>` कौम्पोनॅन्ट का `onClick` prop React को क्लिक इवेंट लिंस्टेनेर सेट करने को कहता है `

How about this?

Reply:
kehta hai doesn't seem right

Reply:
any suggestions? `karta hai`?

Reply:
karta hai seems right

Reply:
reading this again, kehta hai made more sense. Sorry! :(